client/extensions/woocommerce/state/ui/shipping/zones/selectors.js

```diff
@@ -13,6 +13,39 @@ export const getShippingZonesEdits = ( state, siteId ) => {
 	return get( state, [ 'extensions', 'woocommerce', 'ui', 'shipping', siteId, 'zones' ] );
 };
 
+export const orderShippingZones = ( zones ) => {
```

Comment:
It would be good to see a test for the ordering, though I know we are short on time so I wouldn't say it is a blocker. Since there is no test and this looks to only be used in the file as a helper, we can at least drop the export.